X16 Reference - Appendix C - 65C02 Processor.md

```diff
@@ -918,18 +918,18 @@ Stop the Processor
 | --------------- | ----------- | ------ | ------ |
 | Implied         | STP         | 1      | 3      |
 
-Stops the processer and places it in a lower power
-state until a hardware reset occurs. This is mostly
-useful for embedded systems and may have little 
-affect when used with the X16.
+Stops the processor and places it in a lower power
+state until a hardware reset occurs. For the X16 emulator,
+when the debugger is enabled using the `-debug` command-line
+parameter, the STP instruction will break into the debugger
+automatically.
 
 Also the acronym for Stone Temple Pilots, a rock band
```

Comment:
Since Stone Templates didn't come into being until 1989, and the 6502 was developed in the 70s... let's just remove this paragraph completely

Reply:
I've removed the paragraph referencing the Stone Temple Pilots.